client/tracks/index.js

```diff
@@ -84,7 +84,7 @@ const events = {
 		'wcpay_overview_deposits_change_schedule_click',
 	OVERVIEW_TASK_CLICK: 'wcpay_overview_task_click',
 	PAYMENT_DETAILS_VIEW_DISPUTE_EVIDENCE_BUTTON_CLICK:
-		'wcpay_payment_details_view_dispute_evidence_button_click',
+		'wcpay_view_submitted_evidence_clicked',
```

Comment:
We should rename the constant too. 

BUT I really don't think constants are helping here, I'd prefer to see the actual event names in code. 

So… pragmatic steps:

- Minimum - ensure the constant name 100% matches the event name.
- Better but optional – remove the events you're touching from the constants enum, move them inline.

Reply:
Why do I care? Constants are an unnecessary layer of indirection between an event name and how it is used. When I'm investigating why a funnel isn't working, where an event is firing, etc, I want to search the code with the event name (short version, without `wcadmin_`) and see all the relevant UI.

Reply:
>  remove the events you're touching from the constants enum, move them inline.

This is interesting, I've been doing the opposite! I assumed that the enum was the preferred method to help prevent misspelling or something.

Reply:
That is likely to be the intention behind doing it that way (maybe also "move all strings out into a strings file").

Reply:
Another way to look at it: Constants, types etc all make it a little harder to iterate on the event – they assume that we got it perfect upfront.